src/components/_Controllers/CMSPreview.js

```diff
@@ -112,17 +112,19 @@ class CMSPreview extends Component {
     if (!this.state.page[locale]) return <h1>Loading</h1>;
     // Get page data for your specific locale
     const page = this.state.page[locale];
+    const { PREVIEW_CMS_API } = queryString.parse(this.props.location.search);
     return (
       <Switch location={{ pathname: `/${page_type}` }}>
         <Route path="/services" render={props => <Service service={page} />} />
         <Route
           path="/official_document_collection"
           render={props => {
             let collection = page;
+            page.lowerBound = '2018-01-01'
```

Comment:
lowerBound is added to official document collections in the cleanData/static.config process for regular builds. Since this is a preview, I'm setting it to be the default lowest date. Without a date the page crashes.